### PR TITLE
update screenshots docs

### PR DESCRIPTION
**Pull request by lizokm**
Update language in SDK screenshot docs.

### CODE REVIEW

File: docs/platforms/android/enriching-events/screenshots/index.mdx [hunk 1]
```diff
@@ -3,24 +3,22 @@ title: "Screenshots"
...
+Sentry makes it possible to automatically take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink> when a user experiences an error, an exception or a crash.
 
-This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
+This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```
Comment (vivianyentran) [inline docs/platforms/android/enriching-events/screenshots/index.mdx:8]:
```suggestion
This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. This feature is also limited to when taking a screenshot is possible. For example, in some environments, like native iOS, taking a screenshot requires the UI thread, which often isn't available in the event of a crash. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```

Comment (vivianyentran) [reply docs/platforms/android/enriching-events/screenshots/index.mdx:8]: I had a hard time parsing this at first, so hope this adds some clarity

**Review by vivianyentran — APPROVED** [review]

File: docs/platforms/android/enriching-events/screenshots/index.mdx (hunk 1 above)
Comment (lizokm) [reply docs/platforms/android/enriching-events/screenshots/index.mdx:8]: Great suggestion!

File: docs/platforms/apple/common/enriching-events/screenshots/index.mdx [hunk 2]
```diff
@@ -10,28 +10,26 @@ notSupported:
...
+Sentry makes it possible to automatically take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink> when a user experiences an error, an exception or a crash.
 
-This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
+This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```
Comment (lizokm) [inline docs/platforms/apple/common/enriching-events/screenshots/index.mdx:15]:
```suggestion
This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. It's also limited by whether taking a screenshot is possible or not. For example, in some environments, like native iOS, taking a screenshot requires the UI thread, which often isn't available in the event of a crash. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```

File: docs/platforms/dotnet/guides/xamarin/enriching-events/screenshots/index.mdx [hunk 3]
```diff
@@ -3,24 +3,22 @@ title: "Screenshots"
...
+Sentry makes it possible to automatically take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink> when a user experiences an error, an exception or a crash.
 
-This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
+This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```
Comment (lizokm) [inline docs/platforms/dotnet/guides/xamarin/enriching-events/screenshots/index.mdx:8]:
```suggestion
This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. It's also limited by whether taking a screenshot is possible or not. For example, in some environments, like native iOS, taking a screenshot requires the UI thread, which often isn't available in the event of a crash. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```

File: docs/platforms/flutter/enriching-events/screenshots/index.mdx [hunk 4]
```diff
@@ -3,24 +3,22 @@ title: "Screenshots"
...
+Sentry makes it possible to automatically take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink> when a user experiences an error, an exception or a crash.
 
-This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
+This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```
Comment (lizokm) [inline docs/platforms/flutter/enriching-events/screenshots/index.mdx:8]:
```suggestion
This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. It's also limited by whether taking a screenshot is possible or not. For example, in some environments, like native iOS, taking a screenshot requires the UI thread, which often isn't available in the event of a crash. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```

File: docs/platforms/javascript/guides/electron/enriching-events/screenshots/index.mdx [hunk 5]
```diff
@@ -3,24 +3,22 @@ title: "Screenshots"
...
+Sentry makes it possible to automatically take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink> when a user experiences an error, an exception or a crash.
 
-This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
+This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```
Comment (lizokm) [inline docs/platforms/javascript/guides/electron/enriching-events/screenshots/index.mdx:8]:
```suggestion
This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. It's also limited by whether taking a screenshot is possible or not. For example, in some environments, like native iOS, taking a screenshot requires the UI thread, which often isn't available in the event of a crash. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```

File: docs/platforms/kotlin-multiplatform/enriching-events/screenshots/index.mdx [hunk 6]
```diff
@@ -3,13 +3,13 @@ title: "Screenshots"
...
+Sentry makes it possible to automatically take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink> when a user experiences an error, an exception or a crash.
 
-This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
+This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```
Comment (lizokm) [inline docs/platforms/kotlin-multiplatform/enriching-events/screenshots/index.mdx:8]:
```suggestion
This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. It's also limited by whether taking a screenshot is possible or not. For example, in some environments, like native iOS, taking a screenshot requires the UI thread, which often isn't available in the event of a crash. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```

File: docs/platforms/react-native/enriching-events/screenshots/index.mdx [hunk 7]
```diff
@@ -3,26 +3,24 @@ title: "Screenshots"
...
+Sentry makes it possible to automatically take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink> when a user experiences an error, an exception or a crash.
 
-This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
+This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```
Comment (lizokm) [inline docs/platforms/react-native/enriching-events/screenshots/index.mdx:8]:
```suggestion
This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. It's also limited by whether taking a screenshot is possible or not. For example, in some environments, like native iOS, taking a screenshot requires the UI thread, which often isn't available in the event of a crash. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```

File: docs/platforms/unity/enriching-events/screenshots/index.mdx [hunk 8]
```diff
@@ -3,24 +3,22 @@ title: "Screenshots"
...
+Sentry makes it possible to automatically take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink> when a user experiences an error, an exception or a crash.
 
-This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
+This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```
Comment (lizokm) [inline docs/platforms/unity/enriching-events/screenshots/index.mdx:8]:
```suggestion
This feature is only available for SDKs with a user interface, like the ones for mobile and desktop applications. It's also limited by whether taking a screenshot is possible or not. For example, in some environments, like native iOS, taking a screenshot requires the UI thread, which often isn't available in the event of a crash. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
```

File: docs/platforms/android/enriching-events/screenshots/index.mdx [hunk 9]
```diff
@@ -3,24 +3,22 @@ title: "Screenshots"
...
 
-Screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink> and is an opt-in feature. You can enable it as shown below:
+Because screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink>, they are an opt-in feature. You can enable screenshots as shown below:
 
 <PlatformContent includePath="enriching-events/attach-screenshots" />
 
 ## Viewing Screenshots
 
-A thumbnail of the screenshot shows up at the top of the **Issue Details** page:
+If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the (**Issues**)[https://demo.sentry.io/issues/] page.
```
Comment (lizokm) [inline docs/platforms/android/enriching-events/screenshots/index.mdx:18]:
```suggestion
If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the [**Issues**](https://demo.sentry.io/issues/) page.
```

File: docs/platforms/apple/common/enriching-events/screenshots/index.mdx [hunk 10]
```diff
@@ -10,28 +10,26 @@ notSupported:
...
 
-Screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink> and is an opt-in feature. You can enable it as shown below:
+Because screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink>, they are an opt-in feature. You can enable screenshots as shown below:
 
 <PlatformContent includePath="enriching-events/attach-screenshots" />
 
 ## Viewing Screenshots
 
-A thumbnail of the screenshot shows up at the top of the **Issue Details** page:
+If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the (**Issues**)[https://demo.sentry.io/issues/] page.
```
Comment (lizokm) [inline docs/platforms/apple/common/enriching-events/screenshots/index.mdx:29]:
```suggestion
If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the [**Issues**](https://demo.sentry.io/issues/) page.
```

File: docs/platforms/dotnet/guides/xamarin/enriching-events/screenshots/index.mdx [hunk 11]
```diff
@@ -3,24 +3,22 @@ title: "Screenshots"
...
 
-Screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink> and is an opt-in feature. You can enable it as shown below:
+Because screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink>, they are an opt-in feature. You can enable screenshots as shown below:
 
 <PlatformContent includePath="enriching-events/attach-screenshots" />
 
 ## Viewing Screenshots
 
-A thumbnail of the screenshot shows up at the top of the **Issue Details** page:
+If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the (**Issues**)[https://demo.sentry.io/issues/] page.
```
Comment (lizokm) [inline docs/platforms/dotnet/guides/xamarin/enriching-events/screenshots/index.mdx:18]:
```suggestion
If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the [**Issues**](https://demo.sentry.io/issues/) page.
```

File: docs/platforms/flutter/enriching-events/screenshots/index.mdx [hunk 12]
```diff
@@ -3,24 +3,22 @@ title: "Screenshots"
...
 
-Screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink> and is an opt-in feature. You can enable it as shown below:
+Because screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink>, they are an opt-in feature. You can enable screenshots as shown below:
 
 <PlatformContent includePath="enriching-events/attach-screenshots" />
 
 ## Viewing Screenshots
 
-A thumbnail of the screenshot shows up at the top of the **Issue Details** page:
+If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the (**Issues**)[https://demo.sentry.io/issues/] page.
```
Comment (lizokm) [inline docs/platforms/flutter/enriching-events/screenshots/index.mdx:18]:
```suggestion
If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the [**Issues**](https://demo.sentry.io/issues/) page.
```

File: docs/platforms/javascript/guides/electron/enriching-events/screenshots/index.mdx [hunk 13]
```diff
@@ -3,24 +3,22 @@ title: "Screenshots"
...
 
-Screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink> and is an opt-in feature. You can enable it as shown below:
+Because screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink>, they are an opt-in feature. You can enable screenshots as shown below:
 
 <PlatformContent includePath="enriching-events/attach-screenshots" />
 
 ## Viewing Screenshots
 
-A thumbnail of the screenshot shows up at the top of the **Issue Details** page:
+If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the (**Issues**)[https://demo.sentry.io/issues/] page.
```
Comment (lizokm) [inline docs/platforms/javascript/guides/electron/enriching-events/screenshots/index.mdx:18]:
```suggestion
If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the [**Issues**](https://demo.sentry.io/issues/) page.
```

File: docs/platforms/kotlin-multiplatform/enriching-events/screenshots/index.mdx [hunk 14]
```diff
@@ -21,12 +21,10 @@ This feature is supported for iOS and Android only.
 
 ## Viewing Screenshots
 
-A thumbnail of the screenshot shows up at the top of the **Issue Details** page:
+If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the (**Issues**)[https://demo.sentry.io/issues/] page.
```
Comment (lizokm) [inline docs/platforms/kotlin-multiplatform/enriching-events/screenshots/index.mdx:24]:
```suggestion
If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the [**Issues**](https://demo.sentry.io/issues/) page.
```

File: docs/platforms/react-native/enriching-events/screenshots/index.mdx [hunk 15]
```diff
@@ -3,26 +3,24 @@ title: "Screenshots"
...
+Because screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink>, they are an opt-in feature. You can enable screenshots as shown below:
 
 <PlatformContent includePath="enriching-events/attach-screenshots" />
 
 When `attachScreenshot` is enabled and an error event is captured, the screenshots are transferred from native SDKs to React Native over the bridge. This can negatively impact performance on lower-end devices, however, React Native New Architecture will improve the performance of the transfer, as the data are not serialized.
 
 ## Viewing Screenshots
 
-A thumbnail of the screenshot shows up at the top of the **Issue Details** page:
+If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the (**Issues**)[https://demo.sentry.io/issues/] page.
```
Comment (lizokm) [inline docs/platforms/react-native/enriching-events/screenshots/index.mdx:20]:
```suggestion
If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the [**Issues**](https://demo.sentry.io/issues/) page.
```

File: docs/platforms/unity/enriching-events/screenshots/index.mdx [hunk 16]
```diff
@@ -3,24 +3,22 @@ title: "Screenshots"
...
 
-Screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink> and is an opt-in feature. You can enable it as shown below:
+Because screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink>, they are an opt-in feature. You can enable screenshots as shown below:
 
 <PlatformContent includePath="enriching-events/attach-screenshots" />
 
 ## Viewing Screenshots
 
-A thumbnail of the screenshot shows up at the top of the **Issue Details** page:
+If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the (**Issues**)[https://demo.sentry.io/issues/] page.
```
Comment (lizokm) [inline docs/platforms/unity/enriching-events/screenshots/index.mdx:18]:
```suggestion
If one is available, you'll see a thumbnail of the screenshot when you click on a specific issue from the [**Issues**](https://demo.sentry.io/issues/) page.
```

**Pull request merged**